### DISCUSSION TITLE
WingetUI is now UniGetUI

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#1900] [Show and tell]
### Version 3.1.0 stable already includes the rebrand. 

Dear users and contributors,

WingetUI has evolved into supporting different package managers, not only Winget, leading to the name WingetUI confusing new users, and not representing the main point of this application. Every day I am observing more often users getting confused by WingetUI and winget, which is an expectable situation taking into account the little difference between the two names. Therefore, WingetUI will be soon rebranded.

The new name will be:
# **UniGetUI**

You may see this banner within WingetUI informing of the rebrand.

![image](https://github.com/marticliment/WingetUI/assets/53119851/c43b7abe-c6f6-404b-8187-4795fa148eed)

Thank you for your understanding, and excuse me for the inconveniences this may cause.

New Icon:

![128](https://github.com/marticliment/WingetUI/assets/53119851/8f8d1c78-ed88-4a6c-a4a8-c94784dea961)

**Comment by CanePlayz** [comment]
Totally makes sense, fair change 😄

**Reply by aaronliu0130** [reply]
Feels like it should be under announcements, though

**Reply by marticliment** [reply]
I will post an announcement later this month, I'd like to maintain the feedback one for a while

**Reply by Ivan951236** [reply]
true, it is fair change (Since WingetUI not only downloads apps from Winget)

**Comment by redactedscribe** [comment]
@marticliment In case it matters, something I've noticed for a while is that [WinGet](https://github.com/microsoft/winget-cli) refers to itself as "WinGet" (uppercase 'G'), not "Winget[UI]" as is used in the name of your excellent software. You may want to consider "UniGetUI". Regardless, thanks for the release of v3!

**Reply by marticliment** [reply]
I will take a look at this

**Reply by redactedscribe** [reply]
Not to be pedantic, but is "UI" even necessary any more if stepping away from having "Winget" in the name? "Uniget" / "UniGet" sounds good on its own. However, after a quick search online (and on GitHub), keeping "UI" may be beneficial as there are a few projects of varying sizes out there with the same name (mostly "Uniget"). Capitalisation "UniGetUI" would help differentiate. Something to consider.

**Edit:** To avoid the issue (if you consider it one), maybe "ExeGet" (doesn't seem to be used much anywhere)?

**Reply by rolltidehero** [reply]
I had some free time this morning and first time i've been pain free all week, so I figured I would enjoy the goofiness while I could until reality set back in. <3 

NOTE: Updated list can be found here: https://github.com/marticliment/WingetUI/discussions/1900#discussioncomment-9120983

```
1. Software Acquisition & Customization Kit [SACK] 
2. Legendary Open Vault for Everything [LOVE] 
3. Manage, Install & Launch Frontend / Frameworks [MILF] 
4. Search, Update & Install Tool [SUIT] 
5. Complete Universal Package Installer & Downloader [CUPID] 
6. Package & Open-Source Organizer Program [POOP] 
7. Download & Deploy Operator [DODO] 
8. Totally Useful Repository Database [TURD] 
9. Software Transfer & Deployment [STD] 
10. Software Transfer & Deployment Suite [STDs] 
11. Software Tools & Utility Discovery [STUD] 
12. Complete Repository And Package Suite [CRAPS]
```

**Comment by candrapersada** [comment]
if the name of WingetUI becomes UnigetUI will the settings data be lost?

**Reply by marticliment** [reply]
No, I will make sure to migrate all data and things

**Comment by Otiel** [comment]
In case you haven't already seen, "Uniget" is already used as a name for:
* [Unity3D package manager](https://github.com/SaladLab/UniGet)
* [Container tools package manager](https://github.com/uniget-org/cli)
* [A ChatGPT plugin](https://uniget.com)

The last one doesn't concern me as much, but the first two are package managers as well. This could bring confusion regarding the branding of the app and not help discovery through search engines, even though you keep "UI" in the name.

**Reply by aaronliu0130** [reply]
Maybe "WinManUI"?

**Comment by marticliment** [comment]
> This could bring confusion regarding the branding of the app and not help discovery through search engines

Those managers are quite specific, and they are not included with WingetUI, opposed to winget

**Reply by Otiel** [reply]
True! To clarify my words, I have nothing against renaming WingetUI, on the contrary,  I think this is a good idea. I just wanted to shine on the fact that the "Uniget" name is already used among package managers.

But I agree those are more niche than WingetUI aim to be. 😊

**Comment by gilromano** [comment]
Another option would be "MultiGetUI"

**Comment by hosct** [comment]
Have you defined a timeframe yet, when the rename will occur?

**Comment by dabockster** [comment]
Drop the "UI" from the name going forward. Maybe only "Uniget" or something even better.

**Reply by aaronliu0130** [reply]
I don't think so, two quite popular apps are already called Uniget but nothing's called UnigetUI.

**Comment by Tarek-Hasan** [comment]
How about WinPacMan? :rofl:

**Reply by aaronliu0130** [reply]
I'd say WinMan or WinManUI, since at least two pacmans already exist

**Comment by marticliment** [comment]
Hello, 

I am afraid there has been a confusion, the new name has already been chosen, and it will be UnigetUI

**Comment by iPoetDev** [comment]
Naming is hard. Martin is highly productive and super.

**Reply by aaronliu0130** [reply]
~~<small>Superget!</small>~~

**Comment by DailenG** [comment]
What about Gogeta? Pun playing off the syllables "go" and "get", sounds similar to a slang pronunciation of "go getter" and pays a bit of a homage to the fusion of Goku and Vegeta in Dragon Ball Z when they merged to form Gogeta 😋

**Comment by pizzav-xyz** [comment]
like https://uniget.dev/ ?

**Reply by marticliment** [reply]
no, UnigetUI

**Reply by pizzav-xyz** [reply]
> no, UnigetUI

But is it related to the link I sent?

**Reply by aaronliu0130** [reply]
of course not, why would it be

**Comment by rumplin** [comment]
As a translator I just want to point out, that a lot of strings contain "WingetUI" and already see I'm translating text I translated a few version ago again and over again. Please keep this in mind for future releases.

**Reply by aaronliu0130** [reply]
Wouldn't it be possible to just Ctrl+F every string with "WingetUI" and push that back to tolgee?

**Reply by rumplin** [reply]
It's not just the "WingetUI" text, it stuff like "About", where there was already "About WingetUI" etc.

**Reply by aaronliu0130** [reply]
Yes. What about that prevents me from doing a Ctrl+F replacement of "WingetUI" with "UnigetUI" on that?

**Reply by marticliment** [reply]
I will handle this, you translators won't need to retranslate anything

**Comment by Arrmm** [comment]
I think rather than calling it Win or Uni or Multi - get, why don't you just call it MartiGet?  I think it is simple, precise, and straight to the point.  You don't need UI anymore (as others have mentioned) since this is not the "UI" version of Winget.  It is much more beyond it, and far more comprehensive.

My 2 cents.

**Reply by BrianG61UK** [reply]
You need UI because it is a UI for many command line package managers.

**Reply by ewanm89** [reply]
Technically a command line interface is still a UI, just not a graphical one. But hey, you wanted to be pedandic about why you need UI still.

**Comment by heartacker** [comment]
PORTING TO avaloniaui for linux/mac support,

**Reply by aaronliu0130** [reply]
Pretty pointless. This heavily uses Windows commands and only pip and .net are on other operating systems, for which GUIs already exist.

**Comment by heartacker** [comment]
my op:

`WeGetUI`

`WeGetIT`
?

**Reply by aaronliu0130** [reply]
too much tencent communism

**Comment by heartacker** [comment]
AiGetUI
AiGetIT

**Comment by heartacker** [comment]
IGetUI
ITGetUI

**Comment by Tarek-Hasan** [comment]
It would have been nice, if you could rebrand it AppGet. The Original [AppGet](https://github.com/appget/appget) project was an open source package manager, which Microsoft copied to make WinGet without permission of the original author. You can read the full story [here](https://keivan.io/the-day-appget-died/). This story is very little known even in the tech industry, so I would love to see millions of people constantly reminded through the rebranding of the app that, how malicious Microsoft is.

**Reply by Otiel** [reply]
I knew of this story. I'm not sure this is a good idea for @marticliment to enter in a naming "war" against Microsoft.

**Reply by aaronliu0130** [reply]
I don't see how it'd be a naming war. It'd be at most a tribute.

**Reply by marticliment** [reply]
It would be confusing for users. The rename is to avoid further confusion between winget and wingetui, and therefore reusing an already known name would cause even further confusion

**Comment by marticliment** [comment]
## Quick update
UniGetUI will have a new icon, very similar to the current one, but more consistent with Fluent Guidelines (and the new icon will scale better on smaller sizes)

![store](https://github.com/marticliment/WingetUI/assets/53119851/82487950-8c25-4756-9a66-964d7be66f05)

**Reply by Otiel** [reply]
I like it! New icon on the systray as well? 😊

**Reply by aaronliu0130** [reply]
Seems fun! But is it part of Fluent guidelines to stick the top of the arrow to the top? I feel like it'd look less awkward if it was padded down a bit

**Reply by marticliment** [reply]
> I like it! New icon on the systray as well? 😊

For the moment the outline icon will be maintained

**Reply by marticliment** [reply]
> But is it part of Fluent guidelines to stick the top of the arrow to the top?

This is not specified, but what they recommend is to reuse borders to have external borders as smooth as possible

**Reply by aaronliu0130** [reply]
Maybe we could straighten the top of the arrow? Putting two radial-y borders together doesn't feel like reusing
![mspaint mockup](https://github.com/marticliment/WingetUI/assets/11874211/ddc8c30d-717e-462f-ad43-a55945b29013)

**Reply by marticliment** [reply]
I am doing a second iteration of the icon with this change and roundier borders on the main corners

**Reply by marticliment** [reply]
See the post updated with the new version

**Reply by yrjarv** [reply]
I'm no designer, but to me there is something off about having the blue arrow touch the edge. In my opinion, it might look even better if the arrow was moved a tiny bit down (maybe half the size of each grey band?). But I have to say, the entire remaining process does really improve the UX compared to v2 of WingetUI.

**Reply by marticliment** [reply]
@Otiel, despite what I said yesterday, there WILL be new tray icons inline with UniGetUI new icon:
![image](https://github.com/marticliment/WingetUI/assets/53119851/2581fd07-96bd-49bb-9808-ffcd6c7ba7a2)

**Reply by aaronliu0130** [reply]
@yrjarv See above, Fluent guidelines recommend sharing borders which is why Marti did it.

**Reply by Spacellary** [reply]
Looks great.

**Comment by 00-kat** [comment]
So if this is installed with, for example, `scoop install wingetui`, will there be a need to uninstall then reinstall under a new name to get updates?

**Reply by marticliment** [reply]
Hi @cd-CreepArghhh, I can't really answer this question since I am not the one who maintains the scoop package. I suppose they will provide the update seamlessly, but WingetUI should autoupdate anyways.

**Comment by mekwall** [comment]
@marticliment Have you thought about dropping the UI postfix in the new name? It makes sense with the current name, but with the new name it becomes superfluous and unnecessary imo.

**Reply by marticliment** [reply]
uniget js already a thing, so it would be bad for them.

**Reply by aaronliu0130** [reply]
(this has also been talked about above)

**Comment by rolltidehero** [comment]
Some of these I came up with myself.. others I had A.I help... Pick a number! 😂

```
1. Nimble Utility Transporter [NUT]
2. Nasty Universal Toolkit Suite [NUTS]
3. Super App Collecting Kit [SACK]
4. Brilliant Open-source Neat Executable Retriever [BONER]
5. Lucrative Open Vault Endeavor [LOVE]
6. Let's Install Neat Unix Xenials [LINUX]
7. Marvelous Install Liaison Frontend [MILF]
8. Gnome Install Neat Gizmo Extraordinaire Redistribution [GINGER]
9. Software Acquisition & Customization Kit [SACK]
10. Legendary Open Vault for Everything [LOVE]
11. Manage, Install & Launch Frontend / Frameworks [MILF]
12. Search, Update & Install Tool [SUIT]
13. Complete Universal Package Installer & Downloader [CUPID]
14. Package & Open-Source Organizer Program [POOP]
15. Download & Deploy Operator [DODO]
16. Totally Useful Repository Database [TURD]
17. Software Transfer & Deployment [STD]
18. Software Transfer & Deployment Suite [STDs]
19. Software Tools & Utility Discovery [STUD]
20. Complete Repository And Package Suite [CRAPS]
21. Windowsy App Package Manager [WAP-M]
22. Grand Universal Installer Kit [GUIK] 
23. Nifty App Procurement Tool [NAPT]
24. Pragmatic Universal Collector Kit [PUCK]
25. Handy Universal Utility Suite [HUUS]
26. Randomized App Transmuter [RAT]
27. Quirky Universal Installer Toolkit [QUIT]
28. Windows Ingenious Nifty Graphical Expeditor Terminal [WINGET]
29. Phenomenal Orchestrating Wizard Enabling Retrieval [POWER]
30. Marvelous Application Curator Kit [MACK]
31. Super Handy Incredible Friendly Toolkit [SHIFT]
32. Xenial User Interface Nerd Gadget Experience Technician [XUNGET]
33. Radically Awesome Package Kit Installer Necessity Graphical [RAPKINGUI]
35. Discover Apps Real-Timel [D.A.R.T.]
36. Totally Offbeat Toolbox [T.O.O.T.] 
38. Sketchy Hax0r Utility Pack [S.H.U.P.]
39. Profane Orgy Wizard [P.O.W.]
41. Responsive App Installation Dashboard [R.A.I.D.]
42. Package Management Suite [P.M.S.]
43. Package Management Studio [P.M.S.]
44. Complete Universal Manager [C.U.M.]
```

With me being a South Park & Brickleberry fanatic, I'm sure it won't be hard to figure out the ones that I came up with. :P

**Reply by exarobibliologist** [reply]
This is absolutely hysterical.
STD is, by far, my favorite name for a package manager now, followed (in no specific order) very closely by POOP, TURD, PMS, and CUM.

Don't take this the wrong way, but I think everyone should be thankful that neither of us is in charge of renaming WingetUI. lolol

**Comment by songpola** [comment]
How about **OneGetUI** ? 🤔

**Reply by aaronliu0130** [reply]
OneGet was Microsoft's CLI for Chocolatey.

**Comment by kilitary** [comment]
how about ShutGet?

**Comment by arafatx** [comment]
MaxiGet, just suggestion.

**Comment by itayking1990** [comment]
Hey, any ETA for the new name/icon?

**Reply by marticliment** [reply]
expect about to or three weeks before the first wingetui prerelease, and then four or five weeks more until stable

**Reply by marticliment** [reply]
Here it is: https://github.com/marticliment/WingetUI/releases/tag/3.1.0-alpha0

**Comment by Wang-sx0103** [comment]
It has been two months now. Do you have a specific update time?

**Reply by marticliment** [reply]
As said in the post above, expect about to or three weeks before the first wingetui prerelease, and then four or five weeks more until stable

**Reply by marticliment** [reply]
Here it is: https://github.com/marticliment/WingetUI/releases/tag/3.1.0-alpha0

**Reply by Wang-sx0103** [reply]
I've updated to the latest version and it looks great so far!

**Reply by ZPNRG** [reply]
> As said in the post above, expect about to or three weeks before the first wingetui prerelease, and then four or five weeks more until stable

@marticliment   I'm assuming that this timeline HAS changed, correct?  It has been a month and we haven't even had Alpha 2 or Beta 0 yet.  So, it would seem that we've got to be at least 2 or 3 weeks away from 3.1.0 final/stable release.

**Reply by marticliment** [reply]
Yes, it has changed a bit. I had finals and had to improvise a little

**Reply by ZPNRG** [reply]
Well, I didn't know you were a student.  Finals certainly take priority and I hope they went fairly well.

**Comment by BrianG61UK** [comment]
> Drop the "UI" from the name going forward. Maybe only "Uniget" or something even better.

I disagree. It is, after all, a UI for various already existing package managers like WinGet, PIP, Scoop and Chocolatey.

**Comment by tb-mtg** [comment]
Can UniGet be used on Windows Server 2019 (or any Windows Server versions)?

**Reply by marticliment** [reply]
It should work, but I haven't tested it personally

**Comment by Abdurrahman98XX** [comment]
as a flutter dev, i'd say

- getx
- getIt

both are packages not a software names

**Comment by BlackwellNik** [comment]
I see the widgets are following the same name change, although I would say, in both instances, why not just call them UniGet? WinGetUI made sense, when it was a UI for WinGet, yet now it's grown beyond WinGet, and isn't merely a UI for it. I see that there's another service, somewhat similar, at uniget.dev and a .com which could not be confused with your app. As an alternative, it seems no-one is using OmniGet for anything of this nature, which may be even more suitable.

All told, I'm really liking the app, and it's a huge step forward over the free version of SUMo, which was what I was using until late last year.

**Reply by BrianG61UK** [reply]
IMHO UniGetUI is a good name. It is still only a **UI** for various package managers that can all be used without it.

**Comment by Shivabeach** [comment]
Question: 
I installed Version 3.1.0, and it keeps wanting me to update to Beta 2 from Some Python things. That update repeatedly fails.

So I downloaded Beta 2 and installed it, and now it is updating to version 3.1.0.  And then I get into the loops with Beta 2 again.

What is the current release, Or what is the sequence of versions, and what is Some python things? I am caught in a time tunnel here. Thanks

**Reply by marticliment** [reply]
The current release is 3.0.2, but it keeps failing due to an issue with WinGet that will be fixed in the following days

**Reply by Shivabeach** [reply]
Thanks!

**Comment by killtux** [comment]
WingetUI updated itself to the rebranded "UniGetUI". After that all packages are gone, no updates since days and i dont see any i installed packge except "Microsoft.WinGet.Client"

![image](https://github.com/marticliment/UniGetUI/assets/36768145/1e382e55-bbc6-498f-9f3f-c643aeacc1fc)


cant find any open issue related to that. any idea what may gone wrong with auto-update?


edit:

![image](https://github.com/marticliment/UniGetUI/assets/36768145/bbaa1278-3b1d-4fb4-a11b-15ff6eba5c08)


i noticed in the package manager settings there is a warning "Winget could not get found." since is preinstalled, i am very confused how this could be - and its that only installed app under "installed packages" 🤔

**Reply by marticliment** [reply]
Downgrade to 3.0.2 until I can get this issue fixed

Please follow https://github.com/marticliment/UniGetUI/issues/2438 in order to downgrade WingetUI

**Comment by m-dwork** [comment]
WingetUI made sense because it was offering a UI for winget.

UniGetUI is just a unified package manager that isn't a UI of anything. 

This makes UI redundant. 

Just call it uniget =)

Great software.

**Comment by bakomox** [comment]
i suggest OmniGet

**Comment by bakomox** [comment]
also PowerGet or UltiGet aka ultimate
